openmetadata-ui/src/main/resources/ui/playwright/e2e/Flow/RightEntityPanelFlow.spec.ts

```diff
@@ -1288,83 +1029,73 @@ test.describe('Right Entity Panel - Admin User Flow', () => {
       const tabContent = summaryPanel.locator('.data-quality-tab-container');
 
       // Click on Incidents tab
-      const incidentsTabButton = tabContent
-        .locator('.ant-tabs-tab')
-        .filter({ hasText: /incident/i });
+      await navigateToIncidentsTab(adminPage);
 
-      if (await incidentsTabButton.isVisible()) {
-        await incidentsTabButton.click();
-        await adminPage.waitForSelector('[data-testid="loader"]', {
-          state: 'detached',
-        });
-        // Verify incidents tab content is visible
-        const incidentsTabContent = tabContent.locator(
-          '.incidents-tab-content'
-        );
+      // Verify incidents tab content is visible
+      const incidentsTabContent = tabContent.locator('.incidents-tab-content');
 
-        await expect(incidentsTabContent).toBeVisible();
+      await expect(incidentsTabContent).toBeVisible();
 
-        // Verify incident stats container
-        const incidentStatsContainer = incidentsTabContent.locator(
-          '.incidents-stats-container'
-        );
+      // Verify incident stats container
+      const incidentStatsContainer = incidentsTabContent.locator(
+        '.incidents-stats-container'
+      );
 
-        await expect(incidentStatsContainer).toBeVisible();
+      await expect(incidentStatsContainer).toBeVisible();
 
-        // Verify incident stat cards
-        const newCard = incidentStatsContainer.locator(
-          '.incident-stat-card.new-card'
-        );
-        const ackCard = incidentStatsContainer.locator(
-          '.incident-stat-card.ack-card'
-        );
-        const assignedCard = incidentStatsContainer.locator(
-          '.incident-stat-card.assigned-card'
-        );
+      // Verify incident stat cards
+      const newCard = incidentStatsContainer.locator(
+        '.incident-stat-card.new-card'
+      );
+      const ackCard = incidentStatsContainer.locator(
+        '.incident-stat-card.ack-card'
+      );
+      const assignedCard = incidentStatsContainer.locator(
+        '.incident-stat-card.assigned-card'
+      );
```

Comment:
this is fine for now, but you can directly write this `expect`, instead of fetching first and assigning to variable then checking for visibility.

`await expect(ncidentStatsContainer.locator(
        '.incident-stat-card.new-card'
      )).toBeVisible();`